boulder/src/cli/recipe.rs

```diff
@@ -34,6 +34,23 @@ pub struct Command {
 
 #[derive(Debug, clap::Subcommand)]
 pub enum Subcommand {
+    #[command(about = "Bump a recipe's release")]
+    Bump {
+        #[arg(
+            short,
+            long,
+            default_value = "./stone.yaml",
+            help = "Location to output generated build recipe"
```

Comment:
Nit: Something like this? 

```suggestion
            help = "Location of the recipe file to update"
```